vars/RedHat_8.yml

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: MIT
+---
+
+__crypto_policies_os_supported: true
```

Comment:
The RHEL8 version differ only in the package list. 8.1 and 8.2 have just `crypto-policies` package, while 8.3 and newer have also `crypto-policies-scripts` (from vars/main.yml). Is if fine this way or is there a better way to arrange the files? I considered 8.0 as non-production system so I did not create this symlink, but it might make sense to create that one too for completeness, what do you think?

Reply:
I don't think you need a symlink for 8.0
I think you should define the defaults for redhat 8.3 and later in `vars/RedHat_8.yml`, then override those defaults in `vars/RedHat_8.1.yml` and `vars/RedHat_8.2.yml`

Reply:
ah, nevermind - I see you have defined the default `__crypto_policies_packages` in `vars/main.yml` - that's fine